harness/determined/cli/render.py

```diff
@@ -187,15 +189,22 @@ def yes_or_no(prompt: str) -> bool:
         return False
 
 
+def _coloring_enabled() -> bool:
+    return sys.stdout.isatty() and os.environ.get("DET_CLI_COLORIZE", "").lower() in ("1", "true")
```

Comment:
Someday I'd love to see us start using Dynaconf for configuration stuff.

https://www.dynaconf.com/

Not today, but someday...

Reply:
that looks cool, I am looking for something to use for my personal projects 👀

Reply:
do we have/want a general way of "running in dev mode" ?